.github/workflows/lint-test.yml

```
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [16]
+        node-version: [18.x]
```

this should match what "engines" declares (given that GHA always picks the latest release)
```suggestion
        node-version: [18.x, 20.x]
```

---

Seems like `20.x` is blocked until `agoric-sdk` updates its own engines: 
```bash
error @agoric/sdk@: The engine "node" is incompatible with this module. Expected version "^16.13 || ^18.12". Got "20.11.0"
``` 

Do we feel `engines` is adding value in this repo? Would it be better to inherit from `agoric-sdk`?

---

Ah, so it is. Let's leave 20.x out for now.

> Would it be better to remove this field and inherit from agoric-sdk?

Good idea. These doesn't have any narrower constraints, tmk